tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/action/WatchTaskAction.java

```diff
@@ -81,7 +81,9 @@ public void run(IAction action) {
 
 			IProject project = (IProject)element;
 
-			String jobName = project.getName() + ":" + LiferayGradleCore.LIFERAY_WATCH;
+			String jobName =
```

Comment:
Hey @simonjhy  @jtydhr88 

This is not a good design here.  There is called 'leaky abstraction'.  You see that we are creating a 'jobName' which means this method in WatchTaskAction "knows" how other classes (WatchJob) stores jobs by name (using the pattern project:watch:suffix) and then later queries against, and then creates a "new watchjob" later.  This should be addressed in a followup PR.